src/App.vue

```diff
@@ -967,4 +987,28 @@ function getMultilines() {
     .filter((line) => /\w/.test(line)) // removes any empty lines
     .map((line) => line.replace(/\b\n+|\n+\b/g, ""));
 }
+
+// the function of this code is to ai to help select a relevant template -SueAnn Van Valkenburg
+async function selectTemplate() {
+  const userPrompt = this.formData.prompt;
+  console.log("Prompt Text:", userPrompt);
+  try {
+    if (userPrompt != "") {
+      this.inputError = false;
+      const selectedTemplateText = await aiSelectTemplate(userPrompt);
+      console.log(`template : ${selectedTemplateText}`);
+
+      this.formData.input = selectedTemplateText;
+    } else {
+      this.inputError = true;
+    }
+  } catch (error) {
+    console.error("Error:", error);
+  }
+}
 </script>
+<style>
+.error-outline {
+  border-color: red;
+}
+</style>
```

Comment:
Can we just add this  to our CSS file instead of creating an inline style tag here?

Reply:
Ohh yes of course I forgot to move it.